### PR TITLE
[CNFT1-2855] full clean up of Search and base component styles

### PR DESCRIPTION
**Pull request by adamloup-enquizit**
## Description

When fixing the styling issues that were causing the double scroll bars on Investigation search I noticed that the search screen were not all using the same component for the [Accordion](https://github.com/CDCgov/NEDSS-Modernization/blob/42e7988b45bd2b7fb6e4f9ec77137b81e904c339/apps/modernization-ui/src/components/Accordion/Accordion.tsx). after changing to the new component the band aid styling applied to fix specific issues on pages became more and more evident.  I went through each search screen and component removing the copy pasted styles and consolidated them into components or fixed the base component where the issue really existed. 

- All search screens now use the same Accordion component
- Fixed issue of displaying `input` and `select` components within the Accordion
- Created constant for the lists that were using GraphQL enumns to reuse the spread of GraphQL schema types outside of where GraphQL is used.
- Added styling for compact versions of components
  - SingleSelect
  - MultSelect
  - AutocompleteSingle
  - AutocompleteMulti
  - Input
  - PhonenumberInput
  - DatePickerInput
- Removed redundant, unnecessary, and copy pasted styles  
- Changed all components on search screens to use the compact versions
- Fixed the form property names for Provider and Facility on Investigation Search

Search is now using compact components

Patient
image

Investigation
image

Laboratory Report
image

`MultiSelect` is now styled more correctly

image

## Tickets

* [CNFT1-2855](https://cdc-nbs.atlassian.net/browse/CNFT1-2855)
* [CNFT1-2865](https://cdc-nbs.atlassian.net/browse/CNFT1-2865)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

[CNFT1-2855]: https://cdc-nbs.atlassian.net/browse/CNFT1-2855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hclarkEnq — DISMISSED** [review]
This is Great!

**Review by stevegsa — APPROVED** [review]
🚀🚀🚀

File: apps/modernization-ui/src/design-system/select/multi/theme.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Theme } from 'react-select';
+
+const theme = (existing: Theme) => ({
+    ...existing,
+    borderRadius: 2,
+    colors: { ...existing.colors, primary: '#DEE7F5', primary25: '#' }
```
Comment (mpeels) [inline apps/modernization-ui/src/design-system/select/multi/theme.ts:6]: Question: I didn't see `#DEE7F5` in our styles/colors. Should that value be added?

**Comment by mpeels** [comment]
Might i has screenshot?

**Review by hclarkEnq — APPROVED** [review]

**Review by stevegsa — APPROVED** [review]

**Pull request merged**